File: Codeforces/ICPCAssiut/Arrays/ReplaceMinMax/ReplaceMinMax.java
import java.util.Scanner;
public class ReplaceMinMax{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, i, maxPos, minPos;
        long max, min, t;
        n = sc.nextInt();
        long[] arr = new long[n];
        for(i = 0; i < n; i++){
        arr[i] = sc.nextLong();
        }
        max = arr[0];
        maxPos = 0;
        min = arr[0];
        minPos = 0;
        for(i = 0; i < n; i++){
            if(arr[i] > max){
                max = arr[i];
                maxPos = i;
            }
            if(arr[i] < min){
                min = arr[i];
                minPos = i;
            }
        }
        t = arr[maxPos];
        arr[maxPos] = arr[minPos];
        arr[minPos] = t;
        for(i = 0; i < n; i++){
            System.out.print(arr[i] + " ");
        }
        sc.close();
    }
}

File: Codeforces/ICPCAssiut/Arrays/ReplaceMinMax/ReplaceMinMax.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
long long n, i, max, min, maxPos, minPos, t;
cin >> n;
long long arr[n];
for(i = 0; i < n; i++){
    cin >> arr[i];
}
max = arr[0];
maxPos = 0;
for(i = 1; i < n; i++){
    if(max < arr[i]){
        max = arr[i];
        maxPos = i;
    }
}
min = arr[0];
minPos = 0;
for(i = 1; i < n; i++){
    if(min > arr[i]){
        min = arr[i];
        minPos = i;
    }
}
 t = arr[minPos];
   arr[minPos] = arr[maxPos];
   arr[maxPos] = t;

for(i = 0; i < n; i++){
  

    cout << arr[i] << " ";
}
    return 0;
}